setup.cfg

```diff
@@ -14,9 +14,10 @@ classifiers =
 [options]
 packages = ice
 install_requires =
-    Faker
     defopt
+    faker
     fastapi
+    fvalues
```

Comment:
Could you give more Oughters access to the fvalue repo/package so we can make changes if you're on vacation?

Reply:
I'm not familiar with how this stuff works with organisations/teams, it seems pretty weird. I actually had to ask Andreas to make the repo public, it wouldn't let me. Anyway, I added you as an admin and the core team as maintainers.

In https://github.com/oughtinc/fvalues/settings/access it says:

> No base role set. All [Members](https://github.com/orgs/oughtinc/people) can access this repository.

I don't know what 'access' means, given that it's publicly visible.